test/catalog/catalog_test.cpp

```diff
@@ -24,6 +24,7 @@
 #include "storage/storage_manager.h"
 #include "type/ephemeral_pool.h"
 #include "sql/testing_sql_util.h"
+#include "common/timer.h"
```

Comment:
Why do you need to include this header file?

Reply:
This is needed for performance testing. We plan to remove it later.